data/building-hours/8-isl.yaml

```diff
@@ -0,0 +1,17 @@
+name: Office of International Student Life
+image: isl
```

Comment:
```suggestion
# image: isl
```

I worry that the app will crash when it tries to load the image… and unfortunately, we don't support loading images over the network, so we'll have to get the app update ready before we can load the image.